src/test/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/bail/adminofficer/AdminOfficerBailSummaryUploadedPersonalisationTest.java

```diff
@@ -0,0 +1,108 @@
+package uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.bail.adminofficer;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.when;
+
+import java.util.Map;
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.BailCase;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.BailCaseFieldDefinition;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.bail.adminofficer.email.AdminOfficerBailSummaryUploadedPersonalisation;
+import uk.gov.hmcts.reform.iacasenotificationsapi.infrastructure.EmailAddressFinder;
+
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+public class AdminOfficerBailSummaryUploadedPersonalisationTest {
+
+    @Mock
+    BailCase bailCase;
+
+    private Long caseId = 12345L;
+    private String templateId = "someTemplateId";
+    private String bailReferenceNumber = "someReferenceNumber";
+    private String legalRepReference = "someLegalRepReference";
+    private String homeOfficeReferenceNumber = "someHomeOfficeReferenceNumber";
+    private String applicantGivenNames = "someApplicantGivenNames";
+    private String applicantFamilyName = "someApplicantFamilyName";
+
+    @Mock
+    private EmailAddressFinder emailAddressFinder;
+    private AdminOfficerBailSummaryUploadedPersonalisation adminOfficerBailSummaryUploadedPersonalisation;
+
+    @BeforeEach
+    public void setup() {
```

Comment:
Please add a test for the scenario when there is no LR. It should pick appropriate template and not set the legalRepReference field.